### PR TITLE
Display extra group information when expanding a group

### LINKED ISSUE #38: Uitbreiden groepsinformatie in hoofdscherm
Bij iedere groep in het hoofdscherm voegen we een pijltje naar beneden toe. Als je hier op klikt wordt er meer informatie over de groep getoond. 
- ID
- Prefix
- CN

Dit is UI technisch vergelijkbaar met het accepteren van prospects.

schermafbeelding 2016-10-06 om 12 22 35

### PR DESCRIPTION
**Pull request by nicwortel**
Fixes #38 

schermafbeelding 2017-01-24 om 17 36 53

### CODE REVIEW

**Comment by nicwortel** [comment]
@Frankniesten / @hermanvand zie bovenstaande screenshot, komt dat ongeveer overeen met wat jullie in gedachten hadden?

File: src/AppBundle/Model/Group.php [hunk 1]
```diff
@@ -7,14 +7,40 @@
...
+     * An "ad-hoc" group is a group that can be created by everyone, and can for instance be used by a group of
+     * students working on the same project.
+     *
+     * Together with semi-formal groups, ad-hoc groups can also be called "GroupHub groups" because they are created in
+     * GroupHub.
+     *
+     * Please note that the constants were renamed to more closely follow the GroupHub naming, but the values were kept
+     * the same because of legacy issues. The previous name of this constant was TYPE_GROUPHUB.
+     */
+    const TYPE_AD_HOC = 'grouphub';
```
Comment (Tjeerd) [inline src/AppBundle/Model/Group.php]: Nice 👍

File: src/AppBundle/Model/Group.php [hunk 2]
```diff
@@ -175,7 +201,7 @@ public function setType($type)
      */
     public function isOfType($type)
     {
-        Assertion::inArray($type, [self::TYPE_LDAP, self::TYPE_FORMAL, self::TYPE_GROUPHUB]);
+        Assertion::inArray($type, [self::TYPE_FORMAL, self::TYPE_SEMI_FORMAL, self::TYPE_AD_HOC]);
```
Comment (arothuis) [inline src/AppBundle/Model/Group.php]: You could add a more descriptive assertion message here. See the `inArray` function for how the message will be formatted.

File: src/AppBundle/Security/MembershipVoter.php [hunk 3]
```diff
@@ -49,7 +49,7 @@ protected function voteOnAttribute($attribute, $subject, TokenInterface $token)
         }
 
         // A user is allowed to edit his own membership of grouphub groups
-        if ($subject->getType() === Group::TYPE_GROUPHUB) {
+        if ($subject->getType() === Group::TYPE_AD_HOC) {
```
Comment (arothuis) [inline src/AppBundle/Security/MembershipVoter.php]: If this a group, you could use the isOfType function

File: web/css/grouphub.css [hunk 4]
```diff
@@ -886,20 +886,21 @@ li.group_section div:nth-child(2) p:first-child {
...
+li.group_section div:nth-child(2) div.extra-group-info p {
     color: #8a8a8a;
     font-size: 13px;
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
 }
 /* line 49, _group_section.scss */
-li.group_section div:nth-child(2) p:last-child {
+li.group_section div:nth-child(2) p.date_and_owner {
```
Comment (arothuis) [inline web/css/grouphub.css]: I'm glad you removed the other `:last-child` and `:nth-child()` pseudo-selectors in this PR, yet the `:nth-child(2)` that is still present is not really a robust solution. Would be nice to refactor this in due time.

File: web/js/scripts.js [hunk 5]
```diff
@@ -199,6 +199,18 @@ var grouphub = (function ($) {
...
         });
 
+        $groupContainer.on('click', '.toggle_extra_group_info', function () {
+            var $extraGroupInfo = $(this).parents('li').find('.extra-group-info');
+            var $span = $(this).find('span');
+
+            $extraGroupInfo.toggleClass('hidden');
+
+            $span.toggleClass('fa-angle-double-down');
+            $span.toggleClass('fa-angle-double-up');
```
Comment (arothuis) [inline web/js/scripts.js]: Not sure if possible (and if it is more readable), but these two lines could perhaps be shortened to `$span.toggleClass('fa-angle-double-down fa-angle-double-up');`

**Pull request merged**